docs/streamlit_configuration.md

```diff
@@ -226,10 +226,6 @@ token = ""
 
 [deprecation]
 
-# Set to false to disable the deprecation warning for the image format parameter.
```

Comment:
Can you also modify the `last updated` timestamp earlier in this code snippet, to recognize you've changed it?

Reply:
Oh wow, didn't know about that at all. Thanks for the reminder!

(I'm now curious if we can automate this by eg pulling the time of the last Git commit into wherever in the docs this is getting surfaced, or something... maybe a project for when someone has too much time on their hands :P)